### PR TITLE
Update options.md with trpc-options registration

### PR DESCRIPTION
**Pull request by WineChord**
trpc project is going to consume certain option numbers.

### CODE REVIEW

File: docs/options.md [hunk 1]
```diff
@@ -344,3 +344,7 @@ with info about your project (name and website) so we can add an entry for you.
 1. mypy-protobuf
    * Website: https://github.com/nipunn1313/mypy-protobuf
    * Extension: 1151-1154
+
+1. trpc-options
+   * Website: ???
```
Comment (perezd) [inline docs/options.md:349]: We require some linkage to an external website for our documentation. What is TRPC?

Comment (WineChord) [reply docs/options.md:349]: TRPC is a Remote Proceduce Call framework waiting to be open-sourced. In this case, we haven't have a public website url. Is this ok for us to reserve thses option numbers?

Comment (deannagarcia) [reply docs/options.md:349]: We require you have a public website before adding the option number. Update this when you have one.

**Comment by deannagarcia** [comment]
I'm going to close this for now, feel free to reopen or open a new PR when you have a website to add!

**Pull request closed without merging**